omegaconf/grammar/OmegaConfGrammarParser.g4

```diff
@@ -52,10 +52,17 @@ interpolationResolver: INTER_OPEN resolverName COLON sequence? BRACE_CLOSE;
 configKey: interpolation | ID | INTER_KEY;
 resolverName: (interpolation | ID) (DOT (interpolation | ID))* ;  // oc.env, myfunc, ns.${x}, ns1.ns2.f
 
+// Quoted values.
+
+// Ex: "hello world", 'hello ${world}'
+quotedValue: QUOTE_OPEN
+                  (interpolation | ESC | ESC_INTER | SPECIAL_CHAR | ANY_STR)*
+             QUOTE_CLOSE;
```

Comment:
Is the closing quote (single/double) guaranteed to match the opening one?

Reply:
Yes, this is guaranteed by the lexer: it can only generate a `QUOTE_CLOSE` token when it finds the matching quote type (in mode `QUOTED_SINGLE_MODE` when it finds a single quote, and in mode `QUOTED_DOUBLE_MODE` when it finds a double quote). This avoids having to define two different parser rules.